UncomplicatedCustomItems/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="EXILED" version="8.8.1" targetFramework="net48" />
+</packages>
```

Comment:
Delete it. Im will show how to setting this project

Reply:
What's wrong with NuGet?

Reply:
Because the project is configured without working with it

Reply:
Wdym?

Reply:
see .csproj

Reply:
is not relevant is nuget is used or not